contracts/Makefile

```diff
@@ -9,7 +9,7 @@ build: version ## Build contracts.
 	forge build
 
 bindings: build ## Generate golang contract bindings.
-	./bindings/gen.sh OmniPortal FeeOracleV1
+	./bindings/gen.sh OmniPortal FeeOracleV1 TransparentUpgradeableProxy ProxyAdmin
 
 examples-bindings: build ## Generate golang contract bindings for example contracts
 	BINDINGS_DIR=./bindings/examples ./bindings/gen.sh PingPong
```

Comment:
This should not be done here. Ideally, we need a go file named `gen.go` or something like that. Then inside of that file we'd use [a Go generate comment](https://go.dev/blog/generate), for example:
```go
//go:generate ../bindings/gen.sh OmniPortal FeeOracleV1 TransparentUpgradeableProxy ProxyAdmin
```
Then before we need to use the go files, we just run `go generate ./...` to execute all the generate commands through out the project.

Reply:
added a task